datadog/provider_test.go

```diff
@@ -162,10 +162,10 @@ func testAccPreCheck(t *testing.T) {
 	if isReplaying() {
 		return
 	}
-	if v := os.Getenv("DATADOG_API_KEY"); v == "" {
-		t.Fatal("DATADOG_API_KEY must be set for acceptance tests")
+	if v := os.Getenv("DD_API_KEY"); v == "" {
```

Comment:
I believe the CI relies on the secrets being this name. Can we do an `or` and look for the old one as well for now too?

Reply:
Broke it down into its own functions. Let me know what you think:

```
func testAccPreCheck(t *testing.T) {
	if isReplaying() {
		return
	}
	if !isAPIKeySet() {
		t.Fatal("DD_API_KEY must be set for acceptance tests")
	}
	if !isAPPKeySet() {
		t.Fatal("DD_APP_KEY must be set for acceptance tests")
	}
}
```